src/main/java/com/example/solidconnection/university/domain/UnivApplyInfo.java

```diff
@@ -76,7 +77,7 @@ public class UnivApplyInfo {
     @Column(length = 1000)
     private String details;
 
-    @OneToMany(mappedBy = "univApplyInfo", fetch = FetchType.EAGER)
+    @OneToMany(mappedBy = "univApplyInfo", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.EAGER)
     private Set<LanguageRequirement> languageRequirements = new HashSet<>();
 
     @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @OneToMany(mappedBy = "univApplyInfo", cascade = CascadeType.ALL, orphanRemoval = true)
    private Set<LanguageRequirement> languageRequirements = new HashSet<>();

    @ManyToOne(fetch = FetchType.LAZY)
    private University university;
```

회의에서 연관관계는 기본적으로 LAZY로 설정하기로 했었습니다~
[회의에서 결정된 코드](https://github.com/nayonsoso/solid-connect-fork/blob/sandbox/연관관계/src/main/java/com/example/solidconnection/university/domain/UnivApplyInfo.java)

그런데 추천 대학 서비스 원복하시다가 이것까지 포함해서 원복하신 것 같아요.
LAZY로 변경해주세요~
(OneToMany는 디폴트로 LAZY이므로 생략해도 될 것 같습니다.)